lib/bitex_bot/models/api_wrappers/bitstamp/bitstamp_api_wrapper.rb

```diff
@@ -18,7 +18,7 @@ def self.amount_and_quantity(order_id, transactions)
   end
 
   def self.balance
-    balance_summary_parser(Bitstamp.balance.symbolize_keys)
+    balance_summary_parser(Bitstamp.balance(currency_pair).symbolize_keys)
```

Comment:
`balance_summary_parser` doesn't know anything about any currencies other than "btc" and "usd". A spec should have failed, which means this is not being tested or too heavily stubbed.

Reply:
:+1: